web/massastation/src/main.tsx

```diff
@@ -56,8 +57,10 @@ const router = createBrowserRouter(
 
 ReactDOM.createRoot(document.getElementById('root') as HTMLElement).render(
   <React.StrictMode>
-    <QueryClientProvider client={queryClient}>
-      <RouterProvider router={router} fallbackElement={<Error />} />
-    </QueryClientProvider>
+    <IconContext.Provider value={{ size: '32' }}>
```

Comment:
quick question: why did you add back the icon context provider ?

Reply:
because the size was too small when I use npm link, so this way we explicitly set the size to 32 of all icons.